aten/src/ATen/native/quantized/cpu/qconv_prepack.cpp

```diff
@@ -477,6 +478,109 @@ c10::intrusive_ptr<ConvPackedParamsBase<kSpatialDim>> PackedConvWeightsOnednn<
 
 template struct PackedConvWeightsOnednn<2>;
 template struct PackedConvWeightsOnednn<3>;
+
+// Return the packed weight as Mkldnn Tensor
+at::Tensor _qconv_prepack_pt2e(
+    at::Tensor weight, // from CPU backend instead of QuantizedCPU
+    at::Tensor weight_scales, // Weight zero points must be 0 for onednn
+    torch::List<int64_t> input_shape,
```

Comment:
Can we make `input_shape` optional? See `mkldnn_reorder_conv2d_weight` in `MKLDNNConversions.cpp`.

Reply:
Thanks and changed `input_shape` as optional to do weight prepack.